src/mz/src/sql_client.rs

```diff
@@ -70,18 +79,56 @@ impl Client {
         url
     }
 
+    /// This function configures an own .psqlrc-mz file
+    /// to include the '\timing' function every time
+    /// the user executes the `mz sql` command.
+    pub fn configure_psqlrc(&self) -> Result<(), Error> {
+        // Look for the '.psqlrc' file in the home dir.
+        let Some(mut path) = dirs::home_dir() else {
+            return Err(Error::HomeDirNotFoundError);
+        };
+        path.push(PG_PSQLRC_MZ_FILENAME);
+
+        // Check if the file doesn't exists yet
+        // and create it.
+        if !Path::new(&path).exists() {
+            match fs::File::create(&path) {
+                Ok(mut file) => file.write_all(PG_PSQLRC_MZ_DEFAULT_CONTENT.as_bytes())?,
+                Err(err) => return Err(Error::IOError(err)),
+            }
+        }
```

Comment:
There is a minor race here, in that the file might get created between when you checked for it and when you create it. It's generally considered better practice to attempt to atomically create the file if it does not exist, and then simply ignore the "already exists" error that comes back. See the documentation of [`OpenOptions::create_new`](https://doc.rust-lang.org/stable/std/fs/struct.OpenOptions.html#method.create_new) for details.

```suggestion
        match OpenOptions::new().write(true).create_new(true).open(&path) {
            Ok(mut file) => {
                file.write_all(PG_PSQLRC_MZ_DEFAULT_CONTENT.as_bytes());
            }
            Err(e) if e.kind() == io::ErrorKind::AlreadyExists {
                // Do nothing.
            }
            Err(e) => return Err(Error::IOError(e)),
        }
        if !Path::new(&path).exists() {
            match fs::File::create(&path) {
                Ok(mut file) => file.write_all(PG_PSQLRC_MZ_DEFAULT_CONTENT.as_bytes())?,
                Err(err) => return Err(Error::IOError(err)),
            }
        }
```